test/DepthImageToLaserScanTest.cpp

```diff
@@ -36,6 +36,8 @@
 // Bring in gtest
 #include <gtest/gtest.h>
 
+#include <sensor_msgs/image_encodings.hpp>
```

Comment:
we follow the [google style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes), our developer guide only highlight where we differ from it.

-    header of the current file
-    C system files.
-    C++ system files.
-    Other libraries' .h files.
-    Your project's .h files.